keras_cv/models/segmentation/segment_anything/sam_mask_decoder.py

```diff
@@ -0,0 +1,273 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_cv.backend import keras
+from keras_cv.backend import ops
+
+
+@keras.utils.register_keras_serializable(package="keras_cv")
+class MLP(keras.layers.Layer):
```

Comment:
Is this different from `MLPBlock`?

Reply:
We could unite those. The only difference is that the `MLPBlock` has architecture `embedding_dim -> mlp_dim -> embedding_dim` while `MLP` has architecture `input_dim -> [hidden_dim] * (num_layers - 1) -> output_dim`. Looks like a low-hanging fruit, will address in the next commit.

Reply:
Done.